### PR TITLE
Add function for dual signing

### PR DESCRIPTION
**Pull request by Robin1985**
The AppendSignature function can be used to append a SHA256 signature for dual signing

### CODE REVIEW

File: src/app/FakeLib/SignToolHelper.fs [hunk 1]
```diff
@@ -65,6 +65,42 @@ let Sign (toolsPath : string) (parameters : SignParams) (filesToSign : seq<strin
 /// Signs all files in filesToSign with the certification file certFile, 
 /// protected with the password in the file passFile. 
 /// The signtool will be search in the toolPath.
```
Comment (forki) [inline src/app/FakeLib/SignToolHelper.fs]: something look wrong with the xml comments!? that's from the old function, right?

Comment (Robin1985) [inline src/app/FakeLib/SignToolHelper.fs]: yes, sorry they belong to the obsolete function and I forgot moving them after inserting the new code

Comment (forki) [inline src/app/FakeLib/SignToolHelper.fs]: please add new XML comments and let me know. looks like a nice addition.

Comment (Robin1985) [inline src/app/FakeLib/SignToolHelper.fs]: ok, done and created a new pull request so I will close this one, thanks

**Pull request closed without merging**